spec/nylas/message_spec.rb

```diff
@@ -186,6 +186,23 @@
     end
   end
 
+  describe "update_folder" do
+    it "move message to new `folder`" do
+      folder_id = "9999"
```

Comment:
Is this number Magic?

Would `rand(100)` also work? I'm not sure if this is a great idea, curious to hear what you think.

Reply:
We don't need to use random here but we can use if we want as we are going to assert `folder_id` anyway in the expectation.

We have been using these magic numbers from a long time.